_data/team.yml

```diff
@@ -1,22 +1,22 @@
-Presidents:
-  - image: rosalind
-    name: Rosalind Lutsky
-    position: Co-President
-    year: Junior
-    email: rlutsky@stanford.edu
-    artist: Band of Horses
-    artist_link: https://open.spotify.com/artist/0OdUWJ0sBjDrqHygGUXeCF
+Directors:
   - image: tommy
     name: Tommy Choi
-    position: Co-President
+    position: Co-Director
     year: Junior
-    email: tjchoi@stanford.edu
+    email: tjchoi 
```

Comment:
Sorry, forgot to add an explanation for this! Some of the emails were getting cut off on my browser, and I couldn't figure out any way to fix this without making the font super tiny. I tried out this solution -- removing the @stanford.edu but leaving the click-through links the same -- because I figured it served the same purpose.

This is admittedly a sub-optimal solution. If you have a better idea lmk!